mm2src/coins_activation/src/platform_coin_with_tokens.rs

```diff
@@ -476,7 +476,7 @@ where
     if req.request.tx_history() {
         platform_coin.start_history_background_fetching(
             ctx.clone(),
-            TxHistoryStorageBuilder::new(&ctx).build()?,
+            TxHistoryStorageBuilder::new(&ctx, platform_coin.clone().into().account_db_id().await).build()?,
```

Comment:
In all other calls of TxHistoryStorageBuilder::new() it is called with 'tx_history_db_id()' param and only here it is 'account_db_id()'. Is this correct?

Reply:
good catch! thanks